### ISSUE TITLE
possible bug in help output ?

### ISSUE THREAD

**Issue by colindaven** [nanoporetech/megalodon#4]
Thanks for the interesting tool!

I think there is a "not" missing here:

```
but --variant-filename provided. 
should be
but --variant-filename not provided. 


cat slurm-2909915.out
Input dir:  /ngsssd1/rcug/ONT_thermo_2019/20190620_1055_MN32001_FAK85862_bf499a51/fast5
[09:41:08] Using canoncical alphabet ACGT and modified bases Y=6mA Z=5mC.
[09:41:08] Loading reference.
****************************************************************************************************
        ERROR: per_read_snps output requested, but --variant-filename provided. 
****************************************************************************************************

```

**Comment by marcus1487** [comment]
Yes. This is a bug. Thank you for the report. This fix will be included in the next release.

**Comment by marcus1487** [comment]
Fixed in [1.0.0-alpha.1](https://github.com/nanoporetech/megalodon/tree/1.0.0-alpha.1).
